realm/realm-library/src/main/java/io/realm/internal/LinkView.java

```diff
@@ -153,6 +153,12 @@ public void removeAllTargetRows() {
         nativeRemoveAllTargetRows(nativePointer);
     }
 
+    public Table getTargetTable() {
```

Comment:
Should we cache this? Right now you create a new Table object this method is called.